acrobat/scripts/contentSecurityPolicy/dev.js

```diff
@@ -10,6 +10,7 @@ const connectSrc = [
   '14257-chimera-stage.adobeioruntime.net',
   '14257-chimera.adobeioruntime.net',
   '*.adobe.com',
+  '*.adobeccstatic.com',
```

Comment:
do we need the wildcard? how many hosts are possible?

Reply:
@TsayAdobe do we need the `*`

Reply:
These are the hosts I found in [UniversalNav.js](https://stage.adobeccstatic.com/unav/1.0/UniversalNav.js):

- https://prod.adobeccstatic.com
- https://dev.adobeccstatic.com
- https://stage.adobeccstatic.com

Reply:
if that's the case, we should list each of those hosts under the dev/stage/prod sections, respectively.  no wildcard.

Reply:
Updated with exact hosts